### DISCUSSION TITLE
Transcription

### DISCUSSION THREAD

**Discussion by Symbreaker** [monkeytypegame/monkeytype#4470] [Ideas]
A recording with a playbutton and waveform card, maybe a slow mode too. Speech-to-Text, helping scientists make their databases would be a noble cause. And if the point is to learn to type in different situations, it seems like a fun edition, especially for **court reporters** who might be using monkeytype for **stenography**, or just the language lovers on monkeytype

**Comment by Miodec** [comment]
You can play your sound in any way you want and then just use zen mode

**Comment by Warrobot10** [comment]


**Reply by Miodec** [reply]
The project is open source, but its still up to me to approve or reject features. Please dont encourage people to just work on something without getting the green light from me, because I don't want people to be working on something that I don't think fits the project.

**Reply by Warrobot10** [reply]
Sorry..
